queries.sql

```diff
@@ -1,43 +1,75 @@
--- What animals belong to Melody Pond?
-SELECT animals.name
-FROM animals
-INNER JOIN owners ON animals.owner_id = owners.id
-WHERE owners.full_name = 'Melody Pond';
-
--- List of all animals that are Pokemon
-SELECT animals.name
-FROM animals
-INNER JOIN species ON animals.species_id = species.id
-WHERE species.name = 'Pokemon';
-
--- List all owners and their animals (including those who don't own any animal)
-SELECT owners.full_name, animals.name
-FROM owners
-LEFT JOIN animals ON owners.id = animals.owner_id;
-
--- How many animals are there per species?
-SELECT species.name, COUNT(*) AS animal_count
-FROM species
-LEFT JOIN animals ON species.id = animals.species_id
-GROUP BY species.name;
-
--- List all Digimon owned by Jennifer Orwell
-SELECT animals.name
-FROM animals
-INNER JOIN owners ON animals.owner_id = owners.id
-INNER JOIN species ON animals.species_id = species.id
-WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
-
--- List all animals owned by Dean Winchester that haven't tried to escape
-SELECT animals.name
-FROM animals
-INNER JOIN owners ON animals.owner_id = owners.id
-WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
-
--- Who owns the most animals?
-SELECT owners.full_name, COUNT(*) AS animal_count
-FROM owners
-LEFT JOIN animals ON owners.id = animals.owner_id
-GROUP BY owners.full_name
-ORDER BY animal_count DESC
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name AS last_animal_seen
+FROM visits v
+JOIN vets w ON v.vet_id = w.id
+JOIN animals a ON v.animal_id = a.id
+WHERE w.name = 'Vet William Tatcher'
+ORDER BY v.visit_date DESC
 LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT v.animal_id) AS number_of_animals_seen
+FROM visits v
+JOIN vets s ON v.vet_id = s.id
+WHERE s.name = 'Vet Stephanie Mendez';
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT v.name AS vet_name, COALESCE(s.species_name, 'No specialty') AS specialty
+FROM vets v
+LEFT JOIN specializations s ON v.id = s.vet_id;
```

Comment:
👍 Good catch! Since in the previous comment I requested to update the `specializations` table, it's essential to reflect these changes in your queries as well. Here's what you need to do:

- [ ] Update this query to match the project requirements, considering the changes made to the `specializations` table. Make sure it correctly retrieves the required data based on the updated schema. Additionally, please include a screenshot of this updated query in your PR documentation to provide a clear overview of the changes made.

By ensuring your query aligns with the updated schema, you'll maintain consistency in your project's database structure. Great work, and keep the improvements coming! 👍📊🔄